Formula/lib/libtatsu.rb

```diff
@@ -0,0 +1,35 @@
+class Libtatsu < Formula
+  desc "Library handling the communication with Apple's Tatsu Signing Server (TSS)"
+  homepage "https://libimobiledevice.org/"
+  url "https://github.com/libimobiledevice/libtatsu/releases/download/1.0.3/libtatsu-1.0.3.tar.bz2"
+  sha256 "4f69452d23e50e0ffbe844110e6ab6a900d080e051fbda3b7d595f679dee9bc5"
+  license "LGPL-2.1-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libplist"
```

Comment:
```suggestion
  depends_on "libplist"

  uses_from_macos "curl"
```

Reply:
Fixed in ab9c23ced945a46eefb48bfa32945614af8e787f.